src/pages/AboutUs.jsx

```diff
@@ -1,30 +1,36 @@
+import { Link } from "react-router-dom";
+import Button from "../components/Button";
 
 export default function AboutUsPage() {
   return (
-    <div className="bg-primary flex justify-center h-screen">
-      <section className="flex flex-col justify-center items-center space-y-2 mt-[-300px]">
+    <div className="bg-primary flex-col justify-center">
+      <section className="flex flex-col justify-center items-center space-y-2">
         
-          <h1 className="lg:text-6xl md:text-6xl sm: text-4xl text-center mt-0">
+          <h1 className="lg:text-6xl md:text-6xl sm: text-4xl text-center mt-0 text-bg font-semibold">
             planidea
           </h1>
           
-          <h2 className="text-2xl font-semibold text-center mt-0">
+          <h2 className="text-2xl font-semibold text-center mt-0 text-bg">
             workshops created by people for people
```

Comment:
Just a branding thing,the main page says 'workshops people want', I still like this line so maybe it could go into the <p> copy